protocol/src/main/kotlin/models/Quote.kt

```diff
@@ -0,0 +1,32 @@
+package models
+
+import typeid.TypeID
+import java.time.OffsetDateTime
+
+class Quote private constructor(
+  override val metadata: MessageMetadata,
+  override val data: QuoteData,
+  override var signature: String? = null
+) : Message() {
+
+  companion object {
+    fun create(
+      to: String,
+      from: String,
+      exchangeID: TypeID,
+      data: QuoteData,
+    ): Quote {
+      val id = TypeID(MessageKind.quote.name)
```

Comment:
super nit: looks like other message classes just call `TypeID()` inside of the `MessageMetadata` constructor on line 20 :))